### PR TITLE
[next] Update "Routes Manifest" Error

### PR DESCRIPTION
**Pull request by Timer**
This provides the user with additional troubleshooting steps.

### CODE REVIEW

**Comment by miekd** [comment]
@Timer: Thanks for quick update on this! This would have definitely helped me track down the issue I was dealing with. Excited to see this change go through.

**Review by leo — DISMISSED** [review]

**Review by ijjk — DISMISSED** [review]

File: errors/now-next-routes-manifest.md [hunk 1]
```diff
@@ -2,10 +2,15 @@
...
 
-Check for any build errors in the logs and ensure that the output directory setting is either not changed or is pointing to the location of the `.next` output folder (`distDir`).
+In the Vercel dashboard, open your "Project Settings" and draw attention to "Build & Development Settings":
 
-If you are running `next export` you should **not** need to customize the output directory to `out` since the builder automatically detects `next export` being run and uses the output from it.
+1. Ensure that the "Build Command" setting is not changed, or that it calls `next build`. If this command is not changed but you are seeing this error, double check that your `build` script in `package.json` calls `next build`.
+2. Ensure that the "Output Directory" setting is not changed. This value almost never needs to be configured, and is only necessary if you override `distDir` in `next.config.js`.
+3. For `next export` users: **do not override the "Output Directory"**. Next.js automatically detects what folder you outputted `next export` to.
+
+In rare scenarios, this error message can also be caused by a Next.js build failure (if your "Build Command" accidentially returns a exit code 0).
```
Comment (styfle) [inline errors/now-next-routes-manifest.md]:
```suggestion
In rare scenarios, this error message can also be caused by a Next.js build failure (if your "Build Command" accidentally returns an exit code that is not 0).
```

Is this what you mean?

**Review by styfle — APPROVED** [review]

**Review by mcsdevv — APPROVED** [review]
Awesome work!

**Review by TooTallNate — APPROVED** [review]

**Comment by TooTallNate** [comment]
@ijjk needs to approve.

**Review by ijjk — APPROVED** [review]

**Pull request merged**